DSpace/dspace-angular
# `main` branch synced with `dspace-7.6.1` tag (Angular version)

## References
* Fixes DSpace/DSpace#9616 (for dspace-angular)
* PR for backend: DSpace/DSpace#9647

## Description
This PR shows the behavior of running `git merge -s ours dspace-7.6.1` on the `main` branch.

This command will reset their [common ancestor](https://stackoverflow.com/a/58156381) and is also the recommended way to ["supersede old development history of side branches" according to Git documentation.](https://git-scm.com/docs/merge-strategies#Documentation/merge-strategies.txt-ours-1)
* This action should be done after every minor release from the maintenance branch. It should always be merged back to "main" using the "-s ours" flag.

## How to test
Pretend you are starting with a 7.6.1 release, and attempt an upgrade
1. Checkout 7.6.1 to a branch: `git checkout -b test_upgrade_from_7.6.1 dspace-7.6.1`
2. Upgrade by merging this PR's branch into 7.6.1: `git merge main_synched_with_7.6.1`

Alternatively, see what it looks like in git diff:
* Commit differences: https://github.com/DSpace/dspace-angular/compare/dspace-7.6.1...tdonohue:dspace-angular:main_synched_with_7.6.1
* Code differences: https://github.com/DSpace/dspace-angular/compare/dspace-7.6.1..tdonohue:dspace-angular:main_synched_with_7.6.1


**DO NOT MERGE:  This PR is simply a reference example for what this merger looks like.**
